### PR TITLE
Update GPU operator CI following 4.14 GA

### PR DESCRIPTION
**Pull request by empovit**

### CODE REVIEW

File: ci-operator/jobs/rh-ecosystem-edge/ci-tools-nvidia-gpu-operator/rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-presubmits.yaml [hunk 1]
```diff
@@ -520,6 +520,136 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )4.14-nvidia-gpu-operator-e2e,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (sagidayan) [inline ci-operator/jobs/rh-ecosystem-edge/ci-tools-nvidia-gpu-operator/rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-presubmits.yaml:524]: We dont want this to run always

File: ci-operator/jobs/rh-ecosystem-edge/ci-tools-nvidia-gpu-operator/rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-presubmits.yaml [hunk 2]
```diff
@@ -520,6 +520,136 @@ presubmits:
...
+          secretName: manifest-tool-local-pusher
+      - name: pull-secret
+        secret:
+          secretName: registry-pull-credentials
+      - name: result-aggregator
+        secret:
+          secretName: result-aggregator
+    trigger: (?m)^/test( | .* )4.15-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (sagidayan) [inline ci-operator/jobs/rh-ecosystem-edge/ci-tools-nvidia-gpu-operator/rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-presubmits.yaml:582]: same here

**Review by sagidayan — CHANGES_REQUESTED** [review]

**Comment by empovit** [comment]
/pj-rehearse periodic-ci-rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-4.15-nvidia-gpu-operator-e2e-23-6-x
/pj-rehearse periodic-ci-rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-4.14-nvidia-gpu-operator-e2e-23-6-x
/pj-rehearse periodic-ci-rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-4.14-nvidia-gpu-operator-e2e-23-9-x
/pj-rehearse periodic-ci-rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-4.14-nvidia-gpu-operator-e2e-master
/pj-rehearse periodic-ci-rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-4.15-nvidia-gpu-operator-e2e-23-9-x
/pj-rehearse periodic-ci-rh-ecosystem-edge-ci-tools-nvidia-gpu-operator-main-4.15-nvidia-gpu-operator-e2e-master

**Pull request merged**